### PR TITLE
Respond with Grammar

### PR DESCRIPTION
**Pull request by dns2utf8**
If the user wonders how to use the Grammer, it can now request it with `/grammer`

Depends on https://github.com/coredump-ch/coredumpbot/pull/14

### CODE REVIEW

**Comment by dbrgn** [comment]
First, it's "grammar", not "grammer".

Besides that, do you think this is useful? Shouldn't we instead provide a good `/help`?

We could create the type of `/help` that shows you all commands if you don't send any parameters, but shows the usage and parameters of `status` if used like `/help status`.

**Comment by dns2utf8** [comment]
I would like to extend it later. If the user had caused a InvalidSyntax, the Response could hold the Grammer matching the last valid KeyWord.

This could then be used to extend the `/help`

**Comment by dbrgn** [comment]
Most users don't understand the grammar though :)

**Comment by dns2utf8** [comment]
Then they are outsite the `target audience` :wink:

**Comment by dbrgn** [comment]
For the bot or for the command? I don't agree about the first one. If you mean the second one, at least don't expose the function publically, because too many possible commands will confuse the users :)

File: README.md [hunk 1]
```diff
@@ -25,5 +25,6 @@ Start the Bot like this:
     webcam - Get Picture of Webcam
     subscribe - Subscribe to a Sensor with an Duration
     help - Get Help
+    grammer - Get the Grammer for this Bot
```
Comment (rnestler) [inline README.md]: s/[Gg]rammer/grammar
Maybe you should use spell checking :wink: btw. nouns are written in lower case in English

**Comment by dns2utf8** [comment]
Merged at ebada4908fc491bca

**Pull request closed without merging**